core/lib/zksync_core/src/consensus/tests.rs

```diff
@@ -0,0 +1,86 @@
+use super::*;
+use zksync_concurrency::{ctx, scope};
+use zksync_consensus_executor::testonly::FullValidatorConfig;
+use zksync_consensus_roles::validator;
+use zksync_dal::ConnectionPool;
+use zksync_types::{Address, ProtocolVersionId};
+
+fn latest_protocol_version() -> validator::ProtocolVersion {
+    (ProtocolVersionId::latest() as u32)
+        .try_into()
+        .expect("latest protocol version is invalid")
+}
+
+// In the current implementation, consensus certificates are created asynchronously
+// for the miniblocks constructed by the StateKeeper. This means that consensus actor
+// is effectively just backfilling the consensus certificates for the miniblocks in storage.
+#[tokio::test(flavor = "multi_thread")]
+async fn test_backfill() {
+    zksync_concurrency::testonly::abort_on_panic();
+    let ctx = &ctx::test_root(&ctx::RealClock);
+    let rng = &mut ctx.rng();
+    let pool = ConnectionPool::test_pool().await;
+
+    scope::run!(ctx, |ctx, s| async {
+        // Start state keeper.
+        const OPERATOR_ADDRESS: Address = Address::repeat_byte(17);
```

Comment:
Bikeshedding: IMO, `const`s in the middle of the function body look weird; it's a partial case of [a Clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/items_after_statements) that we don't have enabled.